python/ray/actor.py

```diff
@@ -756,7 +755,12 @@ def make_actor(cls, num_cpus, num_gpus, memory, object_store_memory, resources,
             "methods in the `Checkpointable` interface.")
 
     if max_reconstructions is None:
-        max_reconstructions = 0
+        if ray_constants.direct_call_enabled():
+            # Allow the actor creation task to be resubmitted automatically
+            # by default.
+            max_reconstructions = 3
```

Comment:
if `max_reconstructions` is not set, I think it indicates the actor is not reconstructable, which is not related to whether it's direct actor or not?

Reply:
This is only keeping parity with existing Raylet actors, which reconstruct
the actor task regardless of the setting. We can revisit this once
reconstruction fully implemented for direct call actors too, but making
initial creation controlled by this flag makes sense IMO.

Reply:
Can you add a TODO for this so we don't lose track?

Reply:
Actually I intended for this flag to be kept permanently, unless we want to choose something different as a default?